grails-test-suite-base/src/main/groovy/org/grails/support/MockApplicationContext.java

```diff
@@ -216,7 +217,13 @@ public Map<String, Object> getBeansWithAnnotation(Class<? extends Annotation> an
         }
         return submap;
     }
-    
+
+    @NonNull
+    @Override
+    public <A extends Annotation> Set<A> findAllAnnotationsOnBean(@NonNull String beanName, @NonNull Class<A> annotationType, boolean allowFactoryBeanInit) throws NoSuchBeanDefinitionException {
```

Comment:
Is it intentional that this behavior is introduced in the license header update commit?

Reply:
@osscontributor No, that was not my intention. Must have missed to exclude that change from the commit.